css/nixos-site.css

```diff
@@ -99,6 +101,11 @@ pre {
     content: " [nix-shell]$ ";
 }
 
+/* mobile friendly */
+div.main {
+    word-wrap: break-all;
```

Comment:
`break-all` will break all words eve
n where it mak
es no sense to the read
er.
![image](https://user-images.githubusercontent.com/132835/56909960-60407400-6a77-11e9-8639-a313cc1657a8.png)

The hash that is highlighted is the cause of the breaking. What could instead be done is to add zero-width breaks into the long hash(es) of the page, when they are not already in pre-formatted elements.

Reply:
To further explain, [`break-all`](https://www.w3.org/TR/css-text-3/#word-break-property) is intended more for when typesetting assumes it will be broken at the character boundary, the main example being some styles in the CJK families.